ownership-chain/runtime/src/lib.rs

```diff
@@ -711,12 +711,12 @@ construct_runtime!(
 		Sudo: pallet_sudo = 40,
 
 		// Local pallets
-		LivingAssetsOwnership: pallet_living_assets_ownership = 41,
+		LaosLivingAssetsOwnership: pallet_living_assets_ownership = 41,
 
 		// Frontier
-		Ethereum: pallet_ethereum = 50,
-		EVM: pallet_evm = 51,
-		EVMChainId: pallet_evm_chain_id = 52,
+		LaosEthereum: pallet_ethereum = 50,
+		LaosEVM: pallet_evm = 51,
+		LaosEVMChainId: pallet_evm_chain_id = 52,
 		// DynamicFee: pallet_dynamic_fee = 43,
 		BaseFee: pallet_base_fee = 54,
```

Comment:
BaseFee is also part of frontier, it might be renamed as well in order to be consistent?

Reply:
sgtm